lib/customisation.nix

```diff
@@ -365,6 +378,35 @@ rec {
     else
       mapAttrs mkAttrOverridable pkgs;
 
+  makeAlias =
```

Comment:
Nit: We should call this `mkAlias` for consistency with `mkDerivation`, I think.

Also, I feel that “alias” may be a confusing word in general. `llvmPackages = llvmPackages_21;` is an “alias”, but it’s distinct from both the end‐user `aliases.nix` aliases we have for convenience (like `llvmPackages_latest`), temporary compatibility aliases, and from removed‐package `throw`s. I do not know what a better word would be; maybe having helper functions will reduce the need here.